src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -395,6 +395,12 @@ void testSkipToNext() {
     runTabulationTest("skip_to_next_test");
   }
 
+  @Test
+  @DisplayName("test stopping tabulation after round 2")
```

Comment:
Nit: "test stopping tabulation early", no real value in hard-coding a number in the name at this point.